operator/src/main/resources/example-keycloak.yml

```diff
@@ -1,6 +1,20 @@
-apiVersion: keycloak.io/v2alpha1
+apiVersion: keycloak.org/v2alpha1
 kind: Keycloak
 metadata:
   name: example-kc
 spec:
   instances: 1
+  distConfig:
+    KC_DB: postgres
+    KC_DB_URL_HOST: postgres-db
+    KC_DB_USERNAME: ${secret:keycloak-db-secret:username}
```

Comment:
this functionality is already offered by Kubernetes `envFrom`, I do believe that users are used to the Kubernetes API already and we should provide functionalities with the minimum possible attrition and possibly avoiding new and custom concepts.

Reply:
See the comment above please. I wanted to "hide" the impl details from users – that we're using env vars currently

Reply:
commented above as well 👍